src/classes/RD2_DataMigrationBase_BATCH.cls

```diff
@@ -132,16 +132,20 @@ public abstract class RD2_DataMigrationBase_BATCH implements Database.Batchable<
     public void execute(Database.BatchableContext context, List<npe03__Recurring_Donation__c> rds) {
         setBatchJobId(context);
 
-        RD2_OpportunityEvaluationService.overrideVoidOfOpenOpps = true;
-
+        //enable enhanced Recurring Donations during the batch transaction
         if (!RD2_EnablementService.isRecurringDonations2Enabled && isDryRunMode) {
             RD2_EnablementService.isRecurringDonations2EnabledOverride = true;
         }
 
+        //ensure enhanced RD enablement flag is enabled
         if (!RD2_EnablementService.isRecurringDonations2Enabled) {
             throw new MigrationException(System.Label.RD2_ErrorDataMigrationJobCannotBeRun);
         }
 
+        //do not evaluate Opportunities from the RD trigger during migration
+        //migration batch itself will take care of outstanding Opps
+        RD2_OpportunityEvaluationService.isOverride = true;
```

Comment:
@npsp-reedestockton , @force2b : data migration will skip the standard Opp evaluation service.